### ISSUE TITLE
INFO: api.spotify.com does not support ipv6

### ISSUE THREAD

**Issue by Micaxs** [hrkfdn/ncspot#1063]
Issue:
`api.spotify.com` does not support IPV6 connections, thus after logging in with ncspot and you do not see any "Tracks, Albums, Artists, Playlists, Podcasts, Browse" entries, you might be connecting over IPV6 instead of IPV4, in this case there is a simple solution:

Get the ipv4 address of api.spotify.com:
```sh
ping -4 api.spotify.com
```

Edit your hosts (`/etc/hosts`) file:
```sh
# replace 0.0.0.0 with the IP you got from the ping command above
0.0.0.0 api.spotify.com
```

Hope this helps someone.

**Comment by MakeGamer** [comment]
Exactly. I've spend waaaay too long trying out and finding what causes my NCSPOT to not work at all. Thanks man, you've safed me from going insane. This MUST go in the README.

**Comment by hrkfdn** [comment]
This seems like it should be reported to librespot. Can you confirm it's broken with librespot as well? If so, it should be addressed there I believe.

**Comment by hrkfdn** [comment]
Do you have a log for this? So the login works, but search, etc. doesn't? Maybe something to check out with rspotify, the API client library we use.

**Comment by hrkfdn** [comment]
It must be something else as I can query the API using cURL on my VPS with IPv6 just fine. If you have some logs that would be really helpful :)

**Comment by Micaxs** [comment]
> It must be something else as I can query the API using cURL on my VPS with IPv6 just fine. If you have some logs that would be really helpful :)

its been 2 years, so sadly no. Spotify might've fixed it by now, haven't tested or tried since as no longer forced on ipv6. But in case its still an issue the above would still solve it.
